Makefile

```diff
@@ -21,7 +21,7 @@ endif
 TEST_FORMAT = short-verbose
 endif
 
-PIPELINE_VERSION = 0.21.2
```

Comment:
why do we update this if no new client is generated from it?

Reply:
I'm going to update this and regenerate the client after the depending PR banzaicloud/pipeline#2117 in pipeline get merged.